load_balancer/load_balancer.js

```diff
@@ -320,57 +286,20 @@ try {
 } finally {
 
 }
-var node_queue=[];
-for(var i=0;i<nodes_data.Nodes.length;i++)
-{
-  checkNodeConn(nodes_data.Nodes[i]);
-  function checkNodeConn(node) {
-
-    var https_checkConn ={
-      hostname : node.hostname,
-      port : node.port,
-      path : '/connectionCheck',
-      key : fs.readFileSync('./ssl/key.pem'),
-      cert: fs.readFileSync('./ssl/cert.pem'),
-      rejectUnauthorized:false,
-    };
-
-  var checkConnRequest = https.request(https_checkConn,function(res)
-  {
-    var bodyChunks =[];
-    res.on('data',function(chunk)
-    {
-      bodyChunks.push(chunk);
-    }).on('end',function()
-    {
-      var body = Buffer.concat(bodyChunks);
-      if(body.toString()=='true')
-      {
-        console.log("Added "+node.hostname+":"+node.port+" to queue");
-        node_queue.push(node);
-      }
-    });
-  });
-
-  checkConnRequest.on('error',function(err)
-{
-  console.log("Error connecting to "+node.hostname+":"+node.port);
 
+/* This will update the node queue and working nodes will be added.
+The logger level would be info, when logger.js is integrated. */
+status.selectActiveNodes((workingNodes) => {
+  node_queue.push(...workingNodes);
 });
-checkConnRequest.end();
-
-  }
-}
 
 var job_queue = [];
 if(process.env.mode !== "TESTING")
 {
-  server.listen(nodes_data.host_port.port);
-  console.log("Listening at "+nodes_data.host_port.port);
+  server.listen(nodes_data.load_balancer.port);
```

Comment:
please fix the code quality issues raised here.

Reply:
The check doesn't show any new errors here. Can you tell what the error is?

Reply:
You may need to install the codeclimate plugin on chrome / chromium browser to have the codeclimate issues show up on the PR page.

Reply:
I expect 4 errors here:
1. Opening curly brace does not appear on the same line as controlling statement.
2. Strings must use single quote.
3. Unexpected string concatenation.
4. Infix operators must be spaced.
These are previously existing quality errors, and I haven't tried to fix them. The only reason this line has a change is because the host name property was added there and the object in the configuration file nodes_data_config.json had the property name changed.

Reply:
A few more issues exist. I am not able to copy paste the consolidated list.